### DISCUSSION TITLE
Proxy for MySQL ports?

### DISCUSSION THREAD

**Discussion by meteorjs3000** [nginx-proxy/nginx-proxy#1663] [Q&A]
Is it possible to setup `nginx-proxy` to do for multiple MySQL installations on a single server that it can do for multiple web server installations on a single server?

Eg: `mydomain.com:3306` redirects to `mydomain.com:3306`, `anotherdomain.com:3306` redirects to `anotherdomain.com:3307`

**Comment by wader** [comment]
Have a look at #318

**Comment by speto** [comment]
@meteorjs3000 you can check my https://github.com/jwilder/nginx-proxy/issues/318#issuecomment-421473513 or directly a proof of concept for this issue with a little bit different approach here https://github.com/speto/docker-db-tunnel
